pysb/simulator/scipyode.py

```diff
@@ -191,10 +205,9 @@ def rhs(t, y, p):
         # put together the sensitivity matrix)
         jac_fn = None
         if self._use_analytic_jacobian:
-            species_names = ['__s%d' % i for i in
-                             range(len(self._model.species))]
-
-            jac_matrix = ode_mat.jacobian(species_names)
+            species_symbols = [sympy.Symbol('__s%d' % i)
```

Comment:
This is the jacobian fix. `sympy.Matrix.jacobian` failed when passed strings for the differentiation variables, so I generated actual `Symbol` objects.